ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/BindingVerifier.kt

```diff
@@ -55,6 +56,7 @@ abstract class BindingVerifier {
         }
     }
 
-    abstract fun verifyError()
-    abstract fun verify()
+    var isRelayStateGiven: Boolean = false
+    abstract fun decodeAndVerifyError(): org.w3c.dom.Node
```

Comment:
✏️ You can import `org.w3c.dom.Node` and just use `Node` here (same for line 61)